app/views/categories/index.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= render partial: 'shared/navbar', locals: { title: 'Categories'} %>
+
+<div id="categories" class="cat-container d-flex flex-column">
+  <% if @categories.empty?%>
+    <h3 class='no-cat text-center fw-bold'>No Category Available</h3>
+  <% else %>
+    <% @categories.each do |category| %>
+    <br>
+    <div class='container d-flex flex-row justify-content-between p-3 align-items-center rounded shadow border-dark'>
+      <img src="<%= category.icon %>" alt=<%= category.name %>, class= 'image img-fluid img-thumbnail' />
+      <div class='container-info'>
+        <h2><%= link_to category.name, category_path(category.id), class: 'text-decoration-none fs-3' %></h2>
+        <p class='fw-semibold'>$<%= category.expenses.sum(:amount) %></p>
+      </div>
+      <%= button_to 'Delete', category, method: :delete, data: { turbo: false }, class:'btn btn-outline-danger' %>
+    </div>
+    <% end %>
+  <% end %>
+</div>
```

Comment:
- [ ] your design is fine but according to project requirement, lets try to keep to design guideline like color, space and typography
your design :point_down: 
![Screenshot from 2023-08-18 17-52-29](https://github.com/lugard1/budget-app/assets/66097371/b05126fa-3045-4c08-b5f0-5cf2da794a18)
design guide :point_down: 
![Screenshot from 2023-05-06 13-54-49](https://github.com/lugard1/budget-app/assets/66097371/12864103-2af5-42c0-9ed4-8002b16b78cb)